boring/src/ssl/test/mod.rs

```diff
@@ -1122,8 +1120,7 @@ fn client_set_default_curves_list() {
     let ssl_ctx = SslContextBuilder::new(SslMethod::tls()).unwrap().build();
     let mut ssl = Ssl::new(&ssl_ctx).unwrap();
 
-    ssl.client_set_default_curves_list()
-        .expect("Failed to set curves list. Is Kyber768 missing in boringSSL?")
```

Comment:
nit: Perhaps move "Is Kyber768 missing in boringSSL?" to a comment. Here and below.

Reply:
Made it clear that it would panic if Kyber768 was missing, and I also put these changes in a proper commit prior to the introduction of `setup_accept` and `setup_connect`.